src/components/layout.js

```diff
@@ -33,13 +22,7 @@ FlexBox.defaultProps = {
   display: 'flex'
 };
 
-const Grid = styled('div')(
-  compose(
-    layout,
-    grid,
-    position
-  )
-);
+const Grid = styled('div')(compose(layout, grid, position));
```

Comment:
Looks like some prettier changes are being reversed, did you run `yarn format`?

Reply:
I did! Prettier actually changed it to the new diff.

Reply:
i ask because this has been the common cause of failure for me in CI

```
Checking formatting...
src/components/image.js
src/components/table.js
src/components/typography.js
Code style issues found in the above file(s). Forgot to run Prettier?
```

Reply:
We may need to define a column size in prettier config? I'll look into it.